gegelatilib/CMakeLists.txt

```diff
@@ -20,18 +20,21 @@
 # option, see https://cmake.org/cmake/help/latest/command/set.html.
 set(LIBRARY_TARGET_NAME ${PROJECT_NAME})
 
+include_directories("../lib/JsonCpp")
```

Comment:
Add a short comment before this include (even if it is obvious now, it may be useful in case more libs are added later)